tests/filters.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { test, expect } from '@playwright/test';
+
+// We want to make sure that the ecosystem page has all the filters and they are clickable
+test('can select filters on ecosystem pages', async ({ page }) => {
+    page.goto('/ecosystem/clients/');
+
+    // Check if platform filter works
+    await page.locator("#filter-platform").scrollIntoViewIfNeeded()
+    await page.locator("#filters-overlay").isHidden()
+    await page.locator("#filter-platform-menu").isHidden()
+    await page.locator("#filter-platform").click()
+    await page.locator("#filters-overlay").isVisible()
+    await page.locator("#filter-platform-menu").isVisible()
+
+    // Check if platform filter can be used
+    await page.locator("div.reset-links:nth-child(9) > button:nth-child(1)").click()
```

Comment:
I wonder if `nth-child` is our best option here. Minor changes in the UI could break tests. I didn't make my mind around whether it's a good thing or not.

Reply:
We can probably make it smarter. these are just from the devtools copying the css-selector.